air-quality-ui/e2e/fruit_api.spec.ts

```diff
@@ -1,8 +1,8 @@
-import { expect, test } from '@playwright/test';
```

Comment:
Presumably the fruit was just us investigating playwright? Can we remove all these?

Reply:
Ok sure